templates/components/form/fields_macros.html.twig

```diff
@@ -969,3 +969,14 @@
       {{ helper|raw }}
    </label>
 {% endmacro %}
+
+{% macro buttonField(btlabel, btname, bttype = 'button', btvalue = '', btclass = '', bticon = '', options = {}) %}
```

Comment:
`bt` prefix should be removed. We do not use prefixes for other macros parameters.

```suggestion
{% macro buttonField(label, name, type = 'button', value = '', class = '', icon = '', options = {}) %}
```